pkg/relayer/session/session.go

```diff
@@ -163,92 +165,99 @@ func (rs *relayerSessionsManager) ensureSessionTree(sessionHeader *sessiontypes.
 	return sessionTree, nil
 }
 
-// mapBlockToSessionsToClaim is a MapFn which maps committed blocks to a list of
-// sessions which can be claimed as of that block.
+// forEachBlockClaimSessionsFn returns a new ForEachFn that sends a lists of sessions which
```

Comment:
Can you try to rework this to have bullet points. It's a great comment, which makes sense, but really took a lot of cognitive effort to read & undersand.